sdk/core/azure-common/azure/common/__init__.py

```diff
@@ -3,10 +3,10 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 #--------------------------------------------------------------------------
-from ._version import VERSION as _VERSION
+from ._version import VERSION
```

Comment:
why?

Reply:
The intent is to make the version variable match everywhere. This is the only such instance of `_VERSION` that I've found in the code. I can revert if this would cause a problem.

Reply:
nevermind....

Reply:
A brief chat led to a small revision: 
* This change was introduced to standardize things, and it's OK that we use that (except in cases like this where there is no `__all__` defined) 
* This change should be revered for now because diving into setting up an `__all__` is out of scope. 
* Open an issue where we can use a change in the .toml file in the same folder as the package's setup.py to set overrides for future automation if needed.